SOURCES/etc/xapi.d/plugins/smartctl.py

```diff
@@ -14,32 +14,47 @@ def _list_disks():
     disks = []
     result = run_command(['smartctl', '--scan'])
     for line in result['stdout'].splitlines():
-        if line.startswith('/dev/') and not line.startswith('/dev/bus/'):
-            disks.append(line.split()[0])
+        disks.append(line.split()[0])
     return disks
 
+@error_wrapped
+def _list_raids():
```

Comment:
it looks like you are getting the type. It is not specifically for raids. And I think that this function is not needed because we are already doing the scan in _list_disks(). See my previous comment